ocs_ci/utility/utils.py

```diff
@@ -2067,6 +2063,26 @@ def ceph_health_check_base(namespace=None):
         raise CephHealthException(f"Ceph cluster health is not OK. Health: {health}")
 
 
+def create_ceph_health_cmd(namespace):
+    """
+    Forms the ceph health command
+
+    Args:
+        namespace (str): Namespace of OCS
+
+    Returns:
+        str: ceph health command
+
+    """
+    tools_pod = run_cmd(
+        f"oc -n {namespace} get pod -l 'app=rook-ceph-tools' "
```

Comment:
You can take it from constants 

https://github.com/red-hat-storage/ocs-ci/blob/b5831cfafbf197c6f9043317eb26c616ad48fe2f/ocs_ci/ocs/constants.py#L281

Reply:
done